src/istio/common/zarf.yaml

```diff
@@ -59,3 +108,27 @@ components:
                       echo "Deployment 'pepr-uds-core' does not exist. Skipping restart."
                   fi
               fi
+    # Enable this when ready to switch Core components to use ambient
+    #     after:
+    #       - description: "Ensure istio ambient is enabled for Pepr"
+    #         cmd: "./zarf tools kubectl label namespace pepr-system istio.io/dataplane-mode=ambient --overwrite"
+    #       - description: "Ensure istio-injection is disabled for Pepr"
+    #         cmd: "./zarf tools kubectl label namespace pepr-system istio-injection=disabled --overwrite"
+    #       - description: "Cycle Pepr to refresh connections post-ambient"
+    #         cmd: |
+    #           ./zarf tools kubectl rollout restart -n pepr-system deploy/pepr-uds-core-watcher
+    #           ./zarf tools kubectl rollout restart -n pepr-system deploy/pepr-uds-core
+
+  - name: gateway-api-crds
+    required: true
+    manifests:
+      - name: gateway-api-crds
+        files:
+          # renovate: datasource=github-tags depName=kubernetes-sigs/gateway-api versioning=semver
+          - https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v1.2.1/config/crd/standard/gateway.networking.k8s.io_gatewayclasses.yaml
+          # renovate: datasource=github-tags depName=kubernetes-sigs/gateway-api versioning=semver
+          - https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v1.2.1/config/crd/standard/gateway.networking.k8s.io_httproutes.yaml
+          # renovate: datasource=github-tags depName=kubernetes-sigs/gateway-api versioning=semver
+          - https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v1.2.1/config/crd/standard/gateway.networking.k8s.io_grpcroutes.yaml
+          # renovate: datasource=github-tags depName=kubernetes-sigs/gateway-api versioning=semver
+          - https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v1.2.1/config/crd/standard/gateway.networking.k8s.io_referencegrants.yaml
```

Comment:
I think I may have lead you astray here but looks like our common renovate config may pick these up fine without any comments needed: https://github.com/defenseunicorns/uds-common/blob/main/config/renovate.json5#L63-L75